subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -999,13 +999,28 @@ static int handle_sensor_status(struct bt_mesh_model *model, struct bt_mesh_msg_
 
 		/* Occupancy sensor */
 
-		/* OCC_MODE must be enabled for the occupancy sensors to be
-		 * able to turn on the light:
-		 */
-		if (srv->state == LIGHT_CTRL_STATE_STANDBY &&
-		    !atomic_test_bit(&srv->flags, FLAG_OCC_MODE) &&
-		    !(atomic_test_bit(&srv->flags, FLAG_TRANSITION) &&
-		      !atomic_test_bit(&srv->flags, FLAG_MANUAL))) {
+		if ((srv->state == LIGHT_CTRL_STATE_STANDBY &&
+			/* According to the Mesh Model Specification section
+			 * 6.2.5.6 specifications STANDBY state, and the Auto
```

Comment:
```suggestion
			 * 6.2.5.6 about the STANDBY state, if the Auto
```

Reply:
Same below

Reply:
I do not agree in this change, because it is importent to reference to the specifications STANDBY state and not the STANDBY state in the code.

Reply:
The sentence as it stands makes no sense. What is the "and" related to? The "and" makes no sense. What are you trying to say? "According to section 6.2.5.6: when you are in the specifications STANDBY state and the Auto Occupancy is off, the event should not be processed"?

Reply:
Yes, you have understood it correctly.

Reply:
Then:
```suggestion
			 * 6.2.5.6: When in the specifications STANDBY state, and the Auto
```

Reply:
This is becoming nitpicking, but I will add "When in the".